package.json

```diff
@@ -1,30 +1,39 @@
 {
   "name": "stylelint-config-upleveled",
-  "version": "0.0.1",
-  "description": "",
-  "main": "index.js",
-  "files": [
-    "index.js"
-  ],
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "version": "1.0.1",
+  "description": "UpLeveled Stylelint defaults for styling in CSS, SCSS, Sass, CSS Modules and CSS-in-JS",
   "keywords": [
-    "stylelint",
-    "stylelint-config"
+    "config",
+    "css",
+    "sass",
+    "scss",
+    "stylelint-config",
+    "stylelint"
+  ],
+  "repository": "upleveled/stylelint-config-upleveled",
+  "license": "MIT",
+  "author": "UpLeveled",
+  "contributors": [
+    "Jose Höwer <josehower@gmail.com>",
+    "Karl Horky <karl.horky@gmail.com>"
+  ],
+  "type": "module",
+  "main": "index.cjs",
+  "files": [
+    "index.cjs",
+    "tsconfig.base.json"
   ],
-  "author": "Karl Horky",
-  "license": "ISC",
   "dependencies": {
     "postcss-styled-syntax": "0.4.0",
     "stylelint-config-css-modules": "4.2.0",
-    "stylelint-config-recommended": "11.0.0",
-    "stylelint-config-recommended-scss": "10.0.0"
+    "stylelint-config-recommended-scss": "10.0.0",
+    "stylelint-config-recommended": "11.0.0"
   },
   "devDependencies": {
     "@types/prettier": "2.7.2",
-    "eslint": "8.38.0",
     "eslint-config-upleveled": "4.0.12",
+    "eslint": "8.38.0",
+    "stylelint-config-upleveled": "1.0.1",
```

Comment:
dependencies required to lint the css module example

Reply:
if i remove stylelint config then the example in __tests__/ is not going to be linted properly